src/index.html

```diff
@@ -12,11 +12,54 @@
     <title>Aurator</title>
 </head>
 <body class="app">
-    <img src="img/desktop/naprawa__foto.png" alt="foto">
     <div class="container">
-        <div class="row">
-            <div class="col-sm-12">
-                <h2>Aurator</h2>
+        <div class="top">
+            <div class="nav">
+                <div class="nav-logo">
+                    <img src="img/nav__logo.png" alt="logo">
+                </div>
+                <div class="nav-menu">
+                    <ul class="nav-menu__list d-none d-lg-inline-block">
+                        <li class="d-inline"><a class="nav-link h-txt9" href="#">NAPRAWA I REGENERACJA</a></li>
```

Comment:
Please change it to `Naprawa i regeneracja`
- [ ] lowercase

Reply:
Is it important? We received the texts in this way- it means it will be necessary to rewrite all uppercase texts. If we use `text-transform: uppercase;` it is ensured anyways in case of future proofing.

Reply:
Besides we're supposed to receive new texts on monday, maybe we can put it on hold until then?

Reply:
Ok, we can put it on hold. This is important for search engine.